### PR TITLE
adding golangci-lint to the skipper Dockerfile

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: Dockerfile.node-feature-discovery-operator-build [hunk 1]
```diff
@@ -10,5 +10,6 @@ RUN ["apk", "add", "bash", "make", "docker", "curl", "shadow", "git"]
 RUN go install go.uber.org/mock/mockgen@v0.3.0
 RUN curl -L "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /tmp/kubectl
 RUN install -o root -g root -m 0755 /tmp/kubectl /usr/local/bin/kubectl
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.1
```
Comment (ArangoGutierrez) [inline Dockerfile.node-feature-discovery-operator-build:13]: any reason why not the latest? `1.57.2`

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: Dockerfile.node-feature-discovery-operator-build (hunk 1 above)
Comment (yevgeny-shnaidman) [reply Dockerfile.node-feature-discovery-operator-build:13]: Just to be aligned with the one in the Makefile.

Comment (ArangoGutierrez) [reply Dockerfile.node-feature-discovery-operator-build:13]: I won't be against an extra commit in this PR bumping that version

Comment (yevgeny-shnaidman) [reply Dockerfile.node-feature-discovery-operator-build]: Fixed

**Review by ArangoGutierrez — APPROVED** [review]
I'll leave the lgtm to @marquiz

**Review by marquiz — APPROVED** [review]
/lgtm

**Pull request merged**